### PR TITLE
doc: user docs and clean outdated pages

### PR DESCRIPTION
**Pull request by Windforcer**

### CODE REVIEW

**Review by axelar-docs — APPROVED** [review]

**Comment by Windforcer** [comment]
merging for now so the website is cleaned. We can create a separate PR to address any comments and make changes

File: docs/roles/dev/axelar-cli.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Axelar Network CLI
```
Comment (ggutoski) [inline docs/roles/dev/axelar-cli.md:1]: I think this article needs work but it's fine for now.

File: docs/useful-links.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Useful Links
```
Comment (ggutoski) [inline docs/useful-links.md:1]: Eventually we should ditch this article.  Instead links should be moved to header/footer, or a more context-aware article.

File: sidebars.js [hunk 3]
```diff
@@ -27,8 +27,9 @@ const sidebars = {
       items: [
         {
           type: 'category',
-          label: 'End user',
+          label: 'Satellite user',
```
Comment (ggutoski) [inline sidebars.js:30]: We need a more generic term for "end user".  Satellite is just one of the apps that end users might use in the future.

**Pull request merged**